src/components/Common/SubtitleWithBody.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Subtitle, BodyPrimary } from '../Typography'
+
+const ItemBody = styled(BodyPrimary).attrs({
+  noPaddingTop: true,
+  muted: props => props.bg === 'dark',
+  reverse: props => props.bg === 'dark'
+})`
+  > a {
+    text-decoration: underline;
+  }
+`
+
+const ItemSubtitle = styled(Subtitle).attrs({
+  reverse: props => props.bg === 'dark'
+})`
+  padding-bottom: 0;
+`
+
+const SubtitleWithBody = ({ subtitle, body, bg = 'white' }) => (
+  <React.Fragment>
+    <ItemSubtitle bg={bg}>{subtitle}</ItemSubtitle>
+    <ItemBody bg={bg}>{body}</ItemBody>
+  </React.Fragment>
+)
+
+export { ItemSubtitle, ItemBody }
```

Comment:
@SaraVieira I'm exporting these two in separate since it's useful when using this component with `ReactMarkdown` renderer. I also think it'll be handy for the Right hand side of the modal ([this ticket](https://trello.com/c/dBQYMJfh))